custom/covid/management/commands/add_assignment_cases.py

```diff
@@ -0,0 +1,67 @@
+import uuid
+from xml.etree import cElementTree as ElementTree
+
+from custom.covid.management.commands.update_cases import CaseUpdateCommand
+
+from casexml.apps.case.mock import CaseBlock
+from dimagi.utils.chunked import chunked
+
+from corehq.apps.hqcase.utils import submit_case_blocks
+from corehq.form_processor.interfaces.dbaccessors import CaseAccessors
+
+
+BATCH_SIZE = 100
+DEVICE_ID = __name__ + ".add_assignment_cases"
+
+
+def needs_update(case):
+    return case.get_case_property('is_assigned_primary') != 'yes' or \
+        case.get_case_property('is_assigned_temp') != 'yes'
+
+
+def find_owner_id(case, accessor):
+    checkin_case = accessor.get_case(case.assigned_to_primary_checkin_case_id)
+    return checkin_case.get_case_property('hq_user_id')
+
+
+class Command(CaseUpdateCommand):
+    help = "Creates assignment cases for cases of a specified type in an active location"
+
+    def case_block(self, case, owner_id, assignment_type):
+        case_id = uuid.uuid4().hex
+        return ElementTree.tostring(CaseBlock.deprecated_init(
+            create=True,
+            case_id=case_id,
+            case_type='assignment',
+            owner_id=owner_id,
+            index={'parent': (case.get_case_property('case_type'), case.case_id, "extension")},
+            update={"assignment_type": assignment_type},
+        ).as_xml()).decode('utf-8')
+
+    def update_cases(self, domain, case_type, user_id, location):
+        accessor = CaseAccessors(domain)
+        case_ids = accessor.get_open_case_ids_in_domain_by_type(case_type, owner_ids=[location])
```

Comment:
I think this will look like `owner_ids=[None]` when there is no location which is likely not expected behavior, and I am not sure what it will do.

Reply:
@calellowitz I tested this `None` for the location and the command works as expected (as in does not change any cases because they are not contained in the active location).

Reply:
Is that expected? Typically if you don't pass in an argument like that, it will do it for all locations (like the other commands) rather than none. Otherwise, its a required argument in all but name. In the current system, the command will just silently do nothing if you don't pass a location, which could be confusing.

Reply:
Added this commit that sets `case_ids = []` when `self.location` is `None` because no case ids would be in that active location along with additions to the test for that case. A follow-up is required because this is doing essentially what making a required argument would do.

Reply:
Sounds good. Feel free to merge now, and follow up in another PR. I know these need to be run soon